pyqt-apps/siriushla/as_ap_launcher/killbeam_widgets.py

```diff
@@ -63,24 +73,30 @@ def kill_beam(self):
             return [False, 'Could not read Amplitude Reference PV\n'
                            '(SR-RF-DLLRF-01:mV:AL:REF)!']
 
-        # set Amplitude Increase Rate to Immediately
-        if not self._set_pv('SR-RF-DLLRF-01:AMPREF:INCRATE:S',
-                            RFKillBeamHandler.INCRATE_IMMEDIATELY):
+        # set Amplitude Increase Rate to 50 mV/s and wait
+        self._set_pv('SR-RF-DLLRF-01:AMPREF:INCRATE:S',
+                     RFKillBeamHandler.INCRATE_VALUE)
+
+        if not self._wait_pv('SR-RF-DLLRF-01:AMPREF:INCRATE:S',
+                             RFKillBeamHandler.INCRATE_VALUE):
             return [False, 'Could not set RF Amplitude Increase Rate PV\n'
                            '(SR-RF-DLLRF-01:AMPREF:INCRATE:S)!']
-        _time.sleep(RFKillBeamHandler.TIMEOUT_WAIT)
 
-        # set Amplitude Reference to 60mV and wait for TIMEOUT_ACT seconds
-        if not self._set_pv('SR-RF-DLLRF-01:mV:AL:REF:S', 60):
+        # waiting time
+        waitTime = int((ALRef_init - RFKillBeamHandler.REFMIN_VALUE)/50)
```

Comment:
I think i would be better to name this variable `wait_time` to follow our usual and informal naming convention.

Reply:
done!